lib/models/project.js

```diff
@@ -720,6 +719,8 @@ function ensureInstrumentation(cli, ui) {
     return cli.instrumentation;
   }
 
+  // Instrumentation `require` needs to occur inline due to circular dependencies between Instrumentation => getConfig => Project => Instrumentation. getConfig is used in Project to get the project root.
```

Comment:
sorry for nitpicking but could you wrap break this comment into separate lines, around 80 characters long?